XCast/XCast.json

```diff
@@ -197,7 +197,7 @@
             }
         },
         "setFriendlyName":{
-            "summary": "Sets a name for the implementation",
+            "summary": "Sets a name for the implementation.\n  \n### Events \n\n No Events",
```

Comment:
This API allows application to override the default friendly name value with the friendly name  passed as an argument.

Reply:
Updated changes as per your review comments